include/z64message.h

```diff
@@ -0,0 +1,179 @@
+#ifndef Z64MESSAGE_H
+#define Z64MESSAGE_H
+
+#include "PR/ultratypes.h"
+#include "z64view.h"
+#include "unk.h"
+
+struct Actor;
+struct MessageTableEntry;
+struct OcarinaStaff;
```

Comment:
I don't think such a massive undecompiled system should have its header selectively separated out like this. I think it'd be better to just stick with `#include "ultra64.h"` until all the structs and defines of the message system are much further along, then we can split it up like this. Thoughts?

Reply:
I'm not sure what do you mean by 
> I think it'd be better to just stick with #include "ultra64.h"

Do you mean you would prefer to include `ultra64.h` instead of `PR/ultratypes.h`?

Reply:
I don't mind continuing to split headers now, before the system is decomped (we've done it for other systems before), but I think it would fit better in a separate PR, especially if doing it for view too.

Reply:
Agreed with Elliptic. I don't mind splitting headers out sooner rather than later (even for nondecomped systems), and would agree it would probably be better in its own separate PR. However, since it is already done I kind think it should just stay so we don't have to duplicate work later.